R/utilities.R

```diff
@@ -9,11 +9,15 @@ names(rainfall_conv) = c('A>G', 'T>C', 'C>T', 'G>A', 'A>T', 'T>A', 'A>C', 'T>G',
 #'
 #' @return
 #' @export
-#' @import ssh
 #'
 #' @examples
 get_ssh_session = function(host="gphost01.bcgsc.ca"){
-  session = ssh_connect(host=host)
+  if (!requireNamespace("ssh", quietly = TRUE)) {
```

Comment:
I added `library(ssh)` to the top of the `test_remote.R` script because without it in NAMESPACE this throws an error. Is it possible to also add a step to load the library if it's already installed?

Reply:
Good idea. I suppose we could have it load automatically but I think it's frowned upon to put library() or require() into functions like that.

Reply:
Since this function only needs to be run once per "session" I figured this warning would be fine

Reply:
Oh. It's not printing this as a message?
"The ssh package must be installed to use this functionality"